modules/node_modules/@ncigdc/containers/FrequentMutationsChart.js

```diff
@@ -48,7 +48,7 @@ const FrequentMutationsChartComponent = compose(
 
   const bandWidth = (
     (width - chartMargin.right - chartMargin.left) /
-    (ssms.hits.edges.length + 1) / (showSurvivalPlot ? 2 : 1)
+    (ssms.hits.edges.length + 1) / (showSurvivalPlot ? 1 : 2)
```

Comment:
so, never divide by two, just use `(ssms.hits.edges.length + 1)`, then where `FrequentMutationsChart` is used we just make it the size we actually want, so in cases where there isn't a survivalplot we can just add width 50%